features/Steps/MailSteps.php

```diff
@@ -23,4 +26,19 @@ public function aMailHasBeenSentFromToWith(string $messageType, string $from, st
             $mailobject->getContent()
         );
     }
+
+    /**
+     * @When I wait till there are :count mails in the mailbox
+     */
+    public function iWaitTillThereAreMailsInTheMailbox(int $count): void
+    {
+        $elapsedTime = 0;
+        do {
+            $messagesCount = $this->getMailClient()->getMailCount();
+            if ($messagesCount != $count) {
+                sleep(1);
+                $elapsedTime++;
+            }
+        } while ($this->responseState->getTotalItems() != 1 && $elapsedTime++ < 5);
```

Comment:
Should this not be the `messagesCount` variable?

Reply:
I think with this implementation you will always wait 5 seconds regardless of the message count.